templates/fonts/swift4.stencil

```diff
@@ -43,11 +46,17 @@
 
 // MARK: - Implementation Details
 
-{{accessModifier}} struct FontConvertible {
+{{accessModifier}} struct {{fontType}} {
```

Comment:
☝️ I didn't check myself but I trust you Cmd-F'd for `FontConvertible` in those templates to ensure you didn't forget to replace any instance with `{{fontType}}` right? 🙂

Reply:
We'd catch that with the customname output compilation test.